nncf/experimental/tensorflow/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.experimental.tensorflow.quantization import algorithm as experimental_quantization_algorithm
+from nncf.experimental.tensorflow.patch_tf import patch_tf_operations
+
+patch_tf_operations()
```

Comment:
Could you make a proposal on how we can avoid patching all TensorFlow operations?

Reply:
Thanks for the comment! I don't have a proposal right now but I will think about it. I suggest looking at this as a separate task.

Reply:
It makes sense. Could you file a ticket on it by taking into account our offline discussion?

Reply:
I created a separate ticket (Ref: 79868) to try to avoid wrapping during the first import.